.gitignore

```diff
@@ -1,3 +1,6 @@
 *.xml
 *.iml
 *.DS_Store
+node_modules
+.venv
+.vscode
```

Comment:
No need to ignore the .vscode folder. I purposely added it there to synchronize vscode settings (this is unlike the .idea folder from jetbrains).